code/datums/outfit.dm

```diff
@@ -121,6 +121,7 @@
 	var/datum/sprite_accessory/undershirt = null
 	var/datum/sprite_accessory/underwear = null
 	var/datum/sprite_accessory/socks = null
+	var/datum/sprite_accessory/bra = null
```

Comment:
This can be modularized, and also needs some autodocumentation comment before it (using `/// Your comment here`, or the following for multi-line comments, gets parsed by the DM Language Client after shit ggets compiled and allows us to simply hover over the variable's name to know what it does, by seeing a description of what's going on)
```
/**
 * Your multi-line comment
 * here
 */
```